test/models/piece_test.rb

```diff
@@ -98,6 +98,16 @@ def setup
     assert_equal false, knight.valid_move?((knight.x_coord + 0), knight.y_coord + 2), 'knight should not be able to move directly horizontally/vertically'
   end
 
+  test 'pawn move is valid' do
+    pawn = @game.pawns.first
+    assert_equal true, pawn.valid_move?(pawn.x_coord, (pawn.y_coord + 2)), 'move 2 spaces on y axis'
```

Comment:
This can be written as this for short (use `assert` instead of `assert_equal`):

```
assert pawn.valid_move?(pawn.x_coord, (pawn.y_coord + 2)), 'move 2 spaces on y axis'
```